lib/packages/http/__tests__/githubApi.spec.ts

```diff
@@ -12,7 +12,7 @@ describe("Github", () => {
 
     it("should getAllPhoenixContributors easily", async () => {
 
-        Http.mockImplementationOnce(() => {
+        (Http as any).mockImplementationOnce(() => {
```

Comment:
I am not a proud man.

Reply:
`@ts-ignore` might do it as well.... no need to be puristic :smile: 
I had the idea to create a specific http mocking wrapper, such that mocking http become much easier, e.g. `HttpMock.onGet('/foo/bar').reply(201, { bar: 'foo' })`